modules/md5sum/main.nf

```diff
@@ -0,0 +1,35 @@
+process MD5SUM {
+    tag "$meta.id"
+    label 'process_low'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using bcl-convert. Please use docker or singularity containers."
+    }
+    container "debian:bullseye-slim"
```

Comment:
Maybe for consistency use the ubuntu 20.04 container: https://github.com/nf-core/modules/blob/9aadd9a6d3f5964476582319b3a1c54a3e3fe7c9/modules/untar/main.nf#L6-L8

Reply:
fixed it!